packages/plans-grid/src/plans-grid/style.scss

```diff
@@ -40,13 +25,12 @@
 }
 
 .plans-grid__details-container {
-	@media ( max-width: $plans-grid-max-page-width ) {
+	// -1px is becuase this is a `max-width` media query, instead of the usual `min-width`
+	@media ( max-width: $plans-grid-max-page-width - 1px ) {
 		overflow-x: auto;
 		width: 100%;
 		position: absolute;
 		left: 0;
-
-		@include plans-grid-edge2edge-padding;
```

Comment:
Padding has been moved from `.plans-grid__details-container` to `.plans-details__table` (see changes to `packages/plans-grid/src/plans-details/style.scss` in this PR)